services/openshift/scripts/openshift_provision

```diff
@@ -120,5 +120,9 @@ if [ ! -f ${ORIGIN_DIR}/configured.user ]; then
   echo "[INFO] Adding required roles to openshift-dev and admin user ..."
   oadm policy add-role-to-user basic-user openshift-dev --config=${OPENSHIFT_DIR}/admin.kubeconfig
   oadm policy add-cluster-role-to-user cluster-admin admin --config=${OPENSHIFT_DIR}/admin.kubeconfig
+  oc login https://127.0.0.1:8443 -u openshift-dev -p devel \
+        --certificate-authority=${OPENSHIFT_DIR}/ca.crt &>/dev/null
+  oc new-project test --display-name="OpenShift 3 Sample" \
```

Comment:
@hferentschik I don't think you can create a project called default here - there's already a default project existing, but it's not visible to the openshift-devel user.

$ oc new-project default --display-name="Default project for openshift-dev" --description="This is an example project to demonstrate OpenShift v3"
Error from server: project "default" already exists

But we can make it openshift-dev-project if that's what you'd prefer. And the display name could be "Sample project for openshift-dev"? Does that sound good?